### ISSUE TITLE
How to get point cloud in COLMAP world coordinates

### ISSUE THREAD

**Issue by sontung** [ashawkey/torch-ngp#114]
Hi,
Thanks for sharing this great work. I want to output the point cloud coordinates that each image observes (ro + depth*rd) and visualize them in the same convention of the pose that I get from COLMAP. I am using unnorm depth `depth = torch.sum(weights * z_vals, dim=-1)`. For ro and rd I use:
```
    rays_d = directions @ poses_cm[:, :3, :3]
    rays_o = poses_cm[..., :3, 3] # [B, 3]
    rays_o = rays_o[..., None, :].expand_as(rays_d) # [B, N, 3]
    results['rays_o_colmap'] = rays_o
    results['rays_d_colmap'] = rays_d
``` 
where `poses_cm` is the original c2w matrix that I get from the inverse of COLMAP pose (which is `c2w = np.linalg.inv(m)`)

However, here is the result I got from open3D. Any help would be greatly appreciated.

![Screenshot from 2022-09-30 13-56-23](https://user-images.githubusercontent.com/9626274/193186990-724c44a7-a075-42d0-b611-4acb0e3c047b.png)
